identity-apps-core/apps/recovery-portal/src/main/webapp/self-registration-username-request.jsp

```diff
@@ -1806,6 +1806,18 @@
 
                 return false;
             }
+            if (element.type === 'text' && element.value != null && !element.checkValidity()) {
+                $("#" + error_msg_txt).text("<%=IdentityManagementEndpointUtil.i18n(recoveryResourceBundle,
+                    "Please.enter.valid.input")%>");
+                $("#" + error_msg_element).show();
+                $("#" + element_field).addClass("error");
+                var error_msg_txt_element = document.getElementById(error_msg_txt);
```

Comment:
Shouldn't hash be appended here?

```suggestion
                var error_msg_txt_element = document.getElementById("#" + error_msg_txt);
```

Reply:
No need to put `#` prefix with `getElementById()`.